### DISCUSSION TITLE
Unable to open save file archive

### DISCUSSION THREAD

**Discussion by Ronb1238** [diasurgical/DevilutionX#7508] [Q&A] [answered]
I changed 2 save files from sv to hsv to play on hellfire. Then I transferred the files to the devilution file in appdata. one works great, but the other Pops an "Unable to open save file archive" error message and crashes the game every time I try to move between levels.

**Comment by AJenbo** [comment]
share your files or we have little chance to know what the issue is, also what version did you use?

**Reply by Ronb1238** [reply]
I use version 1.5.3. I can't upload the save file to GitHub because it doesn't support that file type. I tried joining the discord but all the invite links are invalid.

**Reply by AJenbo** [reply]
zip it and then upload it

what invite links are you using?

**Reply by Ronb1238** [reply]
Im using https://discord.com/invite/devilutionx from the github page. it says invite invalid.
And here is the save file in zip
[single_0.zip](https://github.com/user-attachments/files/17619174/single_0.zip)

**Reply by AJenbo** [reply]
seems to work fine, what is your handle?

**Reply by StephenCWills** [reply]
Your save archive is missing a bunch of data.
![image](https://github.com/user-attachments/assets/09bb6a19-bb40-427c-923e-4f9383668e4f)

Normally, if you played through to dlvl 16, it should look more like this.
![image](https://github.com/user-attachments/assets/3de78a03-9183-42a4-8df7-c46542ebd3fd)

Just about the only thing you can do with that save file now is start a new game.

**Reply by Ronb1238** [reply]
The Uploaded file was after I tried to load it with devilutionx. for some reason after the error message it rips apart some data (file size goes from 3,040 to 240 KB).
The following are the 2 save files before any altercations. after changing the file extension name to hsv file 1 works, while the other crashes.
[Saves.zip](https://github.com/user-attachments/files/17621051/Saves.zip)


Btw I got the discord link to work, should we move the discussion to there?

**Reply by AJenbo** [reply]
discord is a good placw to chat, not a good place to track issues

**Reply by StephenCWills** [reply, marked as answer]
On dlvl 7, there is an Illusion Weaver that is walking northeast, but its animation info is invalid (`ticksPerFrame` is zero). It's causing a divide-by-zero error when trying to convert the level.

**Reply by Ronb1238** [reply]
Okay so I went back to dlvl 7, killed all the Illusion Weavers, went to dlvl 8 and to dlvl 7 again to reload the level and it fixed the issue.

When i first went back up to dlvl 7 one appeared at the stairs and started attacking me. I think when I first continued to dlvl 8 he was trying to hit but because I continued to dlvl 8 he couldn't find me and that is what caused the issue (?). 

Did you figure this out, with the mqp editor? and more importantly, how can I make sure that it doesn't happen again?

**Reply by AJenbo** [reply]
No the MPQ editor just let him see if al the expected files are there. The rest was likely done using debugging of the game.

Was the save created with DevilutionX or Diablo?

**Reply by Ronb1238** [reply]
Diablo, I downloaded DevilutionX  In order to transfer the save file to Hellfire.

**Reply by AJenbo** [reply]
I created an issue for it. We'll look in to handling this edge case in future versions.

https://github.com/diasurgical/devilutionX/issues/7509

Thanks for sharing the details

**Reply by Ronb1238** [reply]
Thank you for solving it
